src/components/CustomIcons/RecordNetworkLog.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircle as faCircleReg } from '@fortawesome/free-regular-svg-icons';
+import { faCircle as faCircleSol } from '@fortawesome/free-solid-svg-icons';
```

Comment:
Take a peek at `src/components/IconMapper/dynamicFontAwesomeImports.js` . That's how we reduce the bundle size by only using the required fontawesome icon. I get the sense with your import we would circumvent what Nage created for us.